offer-manager-domain/src/main/groovy/life/qbic/business/persons/affiliation/update/UpdateAffiliationInput.groovy

```diff
@@ -0,0 +1,21 @@
+package life.qbic.business.persons.affiliation.update
+
+import life.qbic.business.exceptions.DatabaseQueryException
+import life.qbic.business.persons.affiliation.Affiliation
+import life.qbic.business.persons.affiliation.AffiliationCategory
+
+/**
+ * The input interface for the Update Affiliation use case.
+ *
+ * @since 1.3.0
+ */
+interface UpdateAffiliationInput {
+
+    /**
+     * Updates an affiliation entry
+     * @param oldAffiliation The affiliation to be updated
+     * @param newAffiliation The new affiliation entry
+     * @throws DatabaseQueryException
+     */
+    void updateAffiliationCategory(Affiliation oldAffiliation, Affiliation newAffiliation)
```

Comment:
I think we can simplify this to only get the id and the category. All the other information does not have to cross the border here/ is not relevant to this entrypoint.